app/code/Magento/MediaGalleryUi/Model/GetAssetDetails.php

```diff
@@ -0,0 +1,102 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\MediaGalleryUi\Model;
+
+use Magento\Framework\Stdlib\DateTime\TimezoneInterface;
+use Magento\MediaGalleryApi\Api\Data\AssetInterface;
+
+/**
+ * Provides asset detail for view details section
+ */
+class GetAssetDetails
+{
+    /**
+     * @var TimezoneInterface
+     */
+    private $dateTime;
+
+    /**
+     * @var GetAssetUsageDetails
+     */
+    private $getAssetUsageDetails;
+
+    /**
+     * @param GetAssetUsageDetails $getAssetUsageDetails
+     * @param TimezoneInterface $dateTime
+     */
+    public function __construct(
+        GetAssetUsageDetails $getAssetUsageDetails,
+        TimezoneInterface $dateTime
+    ) {
+        $this->dateTime = $dateTime;
+        $this->getAssetUsageDetails = $getAssetUsageDetails;
+    }
+
+    /**
+     * Get a piece of asset details
+     *
+     * @param AssetInterface $asset
+     * @return array
+     */
+    public function execute(AssetInterface $asset): array
+    {
+        $details = [
+            [
+                'title' => __('Type'),
+                'value' => __('Asset'),
+            ],
+            [
+                'title' => __('Created'),
+                'value' => $this->formatDate($asset->getCreatedAt())
+            ],
+            [
+                'title' => __('Modified'),
+                'value' => $this->formatDate($asset->getUpdatedAt())
+            ],
+            [
+                'title' => __('Width'),
+                'value' => sprintf('%spx', $asset->getWidth())
+            ],
+            [
+                'title' => __('Height'),
+                'value' => sprintf('%spx', $asset->getHeight())
+            ],
+            [
+                'title' => __('Size'),
+                'value' => $this->formatSize($asset->getSize())
+            ],
+            [
+                'title' => __('Used In'),
+                'value' => $this->getAssetUsageDetails->execute($asset->getId())
+            ]
+        ];
+        return $details;
+    }
+
+    /**
+     * Format image size
+     *
+     * @param int $imageSize
+     * @return string
+     */
+    private function formatSize(int $imageSize): string
+    {
+        return $imageSize === 0 ? '' : sprintf('%sKb', $imageSize / 1000);
```

Comment:
`Kb` is not a valid unit. Also - 1kB is 1024B
https://en.wikipedia.org/wiki/Kilobyte
```suggestion
        return $imageSize === 0 ? '' : sprintf('%sKB', $imageSize / 1024);
```

Reply:
Are you aware that when image size is `333B`, then it'll display:
![image](https://user-images.githubusercontent.com/1639941/89739524-bb865200-da81-11ea-8395-45a2573f3329.png)

But also sprintf can format number of digits? 
Eg. `"%.2f"`

Reply:
What is more - `\Zend\Validator\File\Size::toByteString` offers exactly what you have done there.

Reply:
Thanks, went with `$size === 0 ? '' : sprintf('%.2f KB', $size / 1024);`